### DISCUSSION TITLE
Chromium: CSS Page breaks not respected

### DISCUSSION THREAD

**Discussion by karatekaneen** [gotenberg/gotenberg#582] [Q&A]
Thank you for an awesome application!

When we are creating PDFs we have a small component that forces page breaks. It looks like this:

```html
<div style="float: none !important; display: block">
  <div style="page-break-after: always !important; height: 1px; width: 100%"></div>
</div>
```

In version `7.6.1` it does not work while in version `7.7.2` it does work but then we face the issue with #566. 

My guess is that this is because of Chromium but haven't been able to confirm it yet. Looking for a workaround at the moment.

**Comment by gulien** [comment]
Hello @karatekaneen,

I don’t think it’s related to Gotenberg, but Chromium 😊

Your best option is to look on the web if someone had the same issue as you or for alternative!
